.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -0,0 +1,37 @@
+name: Standard Issue Tempalte
+description: Create an Issue
+title: "[Issue]:"
+labels: ["research"]
+assignees:
+  - kitsushadow
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill this out best you can
+  - type: textarea
+    id: why
+    attributes:
+      label: Why or Acceptance Criteria
+      description: What is the reason we determined we need an issue.
```

Comment:
"we" feels exclusive to anyone but our immediate team. I'd like to strike a more open stance.